docs/concepts/index-patterns.asciidoc

```diff
@@ -126,7 +126,7 @@ pattern:
 
 You can use exclusions to exclude indices that might contain mapping errors.
 To match indices starting with `logstash-`, and exclude those starting with `logstash-old` from
-all clusters having a name starting with `cluster_`, you can use `cluster_*:logstash-*,cluster*:logstash-old*`.
+all clusters having a name starting with `cluster_`, you can use `cluster_*:logstash-*,cluster_*:-logstash-old*`.
```

Comment:
Thank you for your first contribution to the Kibana docs!

Looks like there is an extra dash before logstash:

```suggestion
all clusters having a name starting with `cluster_`, you can use `cluster_*:logstash-*,cluster_*:logstash-old*`.
```

Reply:
Once you make the change and sign the [Contributor Agreement](https://www.elastic.co/contributor-agreement), we'll get this change merged.